docs/sphinx/developers/commit-testing.txt

```diff
@@ -98,3 +101,54 @@ If Ant reports that the build was successful, then there is nothing that
 you need to do. Otherwise, it is helpful if you can provide the command,
 branch name, number of failures at the bottom of the Ant output, and the
 :file:`bio-formats-software-test-\*.log` file.
+
+MATLAB tests
+------------
+.. _MATLAB xUnit Test Framework: http://uk.mathworks.com/matlabcentral/fileexchange/22846-matlab-xunit-test-framework
```

Comment:
Just realised this link needs to be updated to https://github.com/psexton/matlab-xunit